com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/HDMetadata.cs

```diff
@@ -10,19 +10,52 @@ sealed class HDMetadata : ScriptableObject
         [SerializeField]
         HDShaderUtils.ShaderID m_ShaderID;
 
+        [SerializeField]
+        string m_SubTargetGuidString;
+
         [SerializeField]
         bool m_MigrateFromOldCrossPipelineSG; // Keep track from which old SG master node we come from
 
+        [SerializeField]
```

Comment:
The following two additions to HDMetaData are to support the MaterialPostProcessor in reimporting a shadegraph on which a material depends.